js/views/modals/purchase/Purchase.js

```diff
@@ -104,14 +111,17 @@ export default class extends BaseModal {
       selectFirst: true,
       radioStyle: true,
     });
+    // render the moderators so it can start fetching and adding moderator cards
+    this.moderators.render();
+    this.moderators.getModeratorsByID();
```

Comment:
The render has to happen here because getModeratorsByID adds moderators to the DOM, and we don't want to re-fetch them by calling this in the render.

The moderator view could probably be refactored to fix that, but it should be a separate PR.